it/controllers/benefits/accommodation/QualifyingRelocationBenefitsControllerISpec.scala

```diff
@@ -16,51 +16,43 @@
 
 package controllers.benefits.accommodation
 
+import builders.models.IncomeTaxUserDataBuilder.anIncomeTaxUserData
+import builders.models.UserBuilder.aUserRequest
+import builders.models.benefits.AccommodationRelocationModelBuilder.anAccommodationRelocationModel
+import builders.models.benefits.BenefitsViewModelBuilder.aBenefitsViewModel
+import builders.models.mongo.EmploymentCYAModelBuilder.anEmploymentCYAModel
+import builders.models.mongo.EmploymentUserDataBuilder.anEmploymentUserData
 import controllers.benefits.accommodation.routes._
 import controllers.benefits.travel.routes.TravelOrEntertainmentBenefitsController
 import controllers.employment.routes.CheckYourBenefitsController
 import forms.YesNoForm
-import models.User
-import models.benefits.{AccommodationRelocationModel, BenefitsViewModel}
-import models.mongo.{EmploymentCYAModel, EmploymentDetails, EmploymentUserData}
+import models.benefits.AccommodationRelocationModel
+import models.mongo.{EmploymentCYAModel, EmploymentUserData}
 import org.jsoup.Jsoup
 import org.jsoup.nodes.Document
 import play.api.http.HeaderNames
 import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
 import play.api.libs.ws.WSResponse
-import play.api.mvc.AnyContentAsEmpty
-import play.api.test.FakeRequest
 import utils.{EmploymentDatabaseHelper, IntegrationTest, ViewHelpers}
 
 class QualifyingRelocationBenefitsControllerISpec extends IntegrationTest with ViewHelpers with EmploymentDatabaseHelper {
 
-  val employmentId = "001"
-  val taxYearEOY: Int = taxYear - 1
-  val urlEOY = s"$appUrl/$taxYearEOY/benefits/qualifying-relocation?employmentId=$employmentId"
-  val urlInYear = s"$appUrl/$taxYear/benefits/qualifying-relocation?employmentId=$employmentId"
+  private val employmentId = "employmentId"
+  private val taxYearEOY: Int = taxYear - 1
+  private val urlEOY = s"$appUrl/$taxYearEOY/benefits/qualifying-relocation?employmentId=$employmentId"
```

Comment:
the urlEOY/urlInYear can be made into a def like the other specs and just pass the taxYear in

Reply:
That is true. Although I have not looked for such issues and we probably have similar in other places. Not really in the scope.

Reply:
fair enough